demo/atom/spinner/themes/carfactory.scss

```diff
@@ -0,0 +1,3 @@
+@import '../../../../themes/carfactory';
+@import '../../../../components/atom/spinner/src/index.scss';
```

Comment:
Why is SUILoader/index.scss not imported in src/index.scss

If the js is imported implicitly, the scss should be as well IMO

Reply:
Would love to have the loader as an optional component, but will integrate it for now